Documentation/ApiOverview/Seo/PageTitleApi.rst

```diff
@@ -64,11 +68,48 @@ this method you can create your own logic to define the correct title.
 
 Usage example, for example, in an :ref:`Extbase <extbase>` controller:
 
-.. code-block:: php
+..  code-block:: php
+    :caption: EXT:my_extension/Classes/Controller/SomeController.php
+
+    $titleProvider = GeneralUtility::makeInstance(MyOwnPageTitleProvider::class);
+    $titleProvider->setTitle('Title from controller action');
+
+
+Configure the new page title provider to be used in your TypoScript setup:
+
+..  literalinclude:: ExampleSetInController/_setup.typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+Example: Use website title as prefix after page title
+-----------------------------------------------------
+
+If you want to construct titles by prefixing the page title with the website
```

Comment:
I am missing the context: As far as I understand, you add the website title to all pages (except detail pages where a custom title provider is in place). This should be communicated.

Reply:
I feel like this is what I am communicating? Do you have a suggestion on how to improve?

Reply:
To summarize the comments:
So it has to be clear if the new title is added or is replacing, and where it's added (prefix, postfix, instead).
Furthermore it should be *noted* that it might never concern detail pages.